src/nvim/optionstr.c

```diff
@@ -695,6 +696,11 @@ const char *did_set_buftype(optset_T *args)
       || opt_strings_flags(buf->b_p_bt, opt_bt_values, NULL, false) != OK) {
     return e_invarg;
   }
+  // when buftype is set to prompt set the last prompt submit pos to lastline
```

Comment:
hard to understand this sentence. is this a correct rewording?

```suggestion
  // buftype=prompt: set the prompt start position to lastline.
```